src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,173 @@
+import clsx from 'clsx';
+import React, {
+  useId,
+  type ChangeEventHandler,
+  type CSSProperties,
+} from 'react';
+import Label from '../Label';
+import Text from '../Text';
+import styles from './Slider.module.css';
+
+const lowestTenMultiplier = (numbers: number[]) => {
+  let multiplier = 1;
+  while (
+    numbers.some((number) => !Number.isInteger((number * multiplier) % 1))
+  ) {
+    multiplier *= 10;
+  }
+  return multiplier;
+};
```

Comment:
Some docstring with an `@example` could be helpful for future developers

Reply:
Added docstring, thanks!